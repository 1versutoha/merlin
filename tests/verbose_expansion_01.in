["tell","struct","type t = int * string\nlet x : t = 5, \"lol\"\n"]
["tell","end",null]
["seek","exact",{"line":2,"col":4}]
["type","enclosing",{"line":2,"col":4}]
["type","enclosing",{"line":2,"col":4}]
