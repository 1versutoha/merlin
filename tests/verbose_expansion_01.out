["return",false]
["return",true]
["return",{"line":2,"col":20}]
["return",[{"start":{"line":2,"col":4},"end":{"line":2,"col":5},"type":"t"},{"start":{"line":2,"col":4},"end":{"line":2,"col":5},"type":"t"}]]
["return",[{"start":{"line":2,"col":4},"end":{"line":2,"col":5},"type":"int * string"},{"start":{"line":2,"col":4},"end":{"line":2,"col":5},"type":"int * string"}]]
